src/app/containers/ATIAnalytics/params/topicPage/buildParams.js

```diff
@@ -0,0 +1,31 @@
+import { buildATIPageTrackPath } from '../../atiUrl';
+
+export const buildTopicPageATIParams = (
+  pageData,
+  requestContext,
+  serviceContext,
+) => {
+  const { platform, statsDestination } = requestContext;
+  const { atiAnalyticsAppName, service } = serviceContext;
+
+  return {
+    appName: atiAnalyticsAppName,
+    contentType: 'index-category',
+    pageTitle: pageData.title,
+    statsDestination,
+    platform,
+    service,
+  };
+};
+
+export const buildTopicPageATIUrl = (
+  topicPageData,
+  requestContext,
+  serviceContext,
+) => {
+  return buildATIPageTrackPath(
+    buildTopicPageATIParams(topicPageData, requestContext, serviceContext),
+  );
+};
```

Comment:
Can implicit return here
```suggestion
) => buildATIPageTrackPath(
  buildTopicPageATIParams(topicPageData, requestContext, serviceContext),
);
```